README.md

```diff
@@ -1,8 +1,7 @@
 ![nukkit](.github/images/banner.png)
 
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](LICENSE)
-[![Build Status](https://ci.nukkitx.com/job/NukkitX/job/Nukkit/job/master/badge/icon)](https://ci.nukkitx.com/job/NukkitX/job/Nukkit/job/master/)
-![Tests](https://img.shields.io/jenkins/t/https/ci.nukkitx.com/job/NukkitX/job/Nukkit/job/master.svg)
+[![PowerNukkit v1.X](https://github.com/GameModsBR/PowerNukkit/workflows/PowerNukkit%20v1.X/badge.svg?branch=master)](https://github.com/GameModsBR/PowerNukkit/actions?query=branch%3Amaster)
```

Comment:
![image](https://user-images.githubusercontent.com/31969146/73716591-2e3efb00-46f6-11ea-928a-87f6bc365a83.png)

The badge returns not found when the branch parameter is passed

[(https://github.com/GameModsBR/PowerNukkit/workflows/PowerNukkit%20v1.X/badge.svg?branch=master](url)
[(https://github.com/GameModsBR/PowerNukkit/workflows/PowerNukkit%20v1.X/badge.svg](url)